src/processes/quantiles.js

```diff
@@ -0,0 +1,34 @@
+const Process = require('../processgraph/process');
+const Commons = require('../processgraph/commons');
+const Errors = require('../errors');
+const underscore = require('underscore');
+
+// TODO: should we check for invalid input, e.g. probabilites > 1?
```

Comment:
Yes, why not? But in validate, not execute. - or can we add such things to the JSON schema?